Sources/Extensions/Foundation/URLExtensions.swift

```diff
@@ -63,6 +63,20 @@ public extension URL {
 		self = appendingQueryParameters(parameters)
 	}
 
+    /// SwifterSwift: Get value of a query key.
+    ///
+    ///    var url = URL(string: "https://google.com?code=12345")!
+    ///    url.value(for: code) -> "12345"
+    ///
+    /// - Parameter: Key of a query.
+    public func value(for queryKey: String) -> String? {
```

Comment:
Hey @jdisho I'm not sure about this naming here, to me `url.value(for:)` even with queryKey variable name,  doesn't make clear that is a value for a query parameter, seems like can be any value, I don't know, maybe something like `queryParameterValue(for key: String)`, but it has the con that is more verbose, but is more clear what it does.
What do you think? 
Note: I'm not good at naming this suggestion is probably not good naming at all 😂 
Let know your thoughts on that :))

Reply:
I agree, and `value(for:)` is a pretty common API for stuff like NSUserDefaults and collections. Something like `queryValue` or `parameterValue` could be good names too. Good suggestion 👍